cmd/create.go

```diff
@@ -429,6 +434,47 @@ func (cc *createCmd) createFiles(detectedLang *config.DraftConfig, lowerLang str
 	return nil
 }
 
+func (cc *createCmd) detectDefaults(detectedLang *config.DraftConfig, lowerLang string) {
+	if lowerLang == "gradlew" || lowerLang == "gradle" {
+		// read from build.gradle and detect version
+		f, err := os.Open("build.gradle")
+		if err != nil {
+			log.Warn("Unable to read build.gradle, skipping detection")
+			return
+		}
+		defer f.Close()
+		detectedDefaults := make([]config.BuilderVarDefault, 0)
+		scanner := bufio.NewScanner(f)
+		separator := func(c rune) bool { return c == ' ' || c == '=' }
```

Comment:
❤️☕️💡 Observation/suggesstion please: What do you think? can we please add comment please for rune/separator context please? because this will be use for the `fieldFunc` funct for specific reasons.